modules/integration/tests-integration/tests-amqp/src/test/resources/testng.xml

```diff
@@ -75,6 +75,8 @@
 
             <!-- Queue -->
 
+            <class name="org.wso2.mb.integration.tests.amqp.functional.ExclusiveConsumerTestCase"/>
```

Comment:
Can we add the test case in the functional test case section in this xml instead of the "load test cases" section.